kvmagent/kvmagent/plugins/tensorfusion/service.py

```
@@ -0,0 +1,296 @@
+                avail = self._tracker.get_available_memory(pci)
+                raise Exception('insufficient GPU memory on %s: requested %dMB, available %dMB' %
+                                (pci, request.memory_mb, avail))
+
+            worker = self._executor.start(request)
+            try:
+                self._store.add(worker)
+                self._tracker.allocate(pci, worker.device_uuid, worker.allocated_memory_mb)
+            except Exception:
+                try:
+                    self._executor.stop(worker)
+                except Exception as stop_error:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)